test/emmy/tape_test.cljc

```diff
@@ -11,6 +11,7 @@
             [emmy.generators :as sg]
             [emmy.generic :as g]
             [emmy.numerical.derivative :refer [D-numeric]]
+            [emmy.operator :as o]
```

Comment:
I just had this random test idea, which I don't think is particularly valuable or anything, but thought it was cool: "the Jacobian of a permutation is the permutation matrix of that permutation", as in :

```
(defn f [a b c d e] (up d e c b a))
(def M ((D f) 'a 'b 'c 'd 'e))
;; M = (down (up 0 0 0 0 1) (up 0 0 0 1 0) (up 0 0 1 0 0) (up 1 0 0 0 0) (up 0 1 0 0 0))
;; and so:
 (* M [1 2 3 4 5])
;; => (up 4 5 3 2 1)

Reply:
Added as a test!